pydis_site/apps/api/models/bot/user.py

```diff
@@ -86,3 +93,45 @@ def username(self) -> str:
         For usability in read-only fields such as Django Admin.
         """
         return str(self)
+
+
+class UserAltRelationship(ModelReprMixin, ModelTimestampMixin, models.Model):
+    """A relationship between a Discord user and its alts."""
+
+    source = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        verbose_name="Source",
+        help_text="The source of this user to alternate account relationship",
+    )
+    target = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        verbose_name="Target",
+        related_name='+',
+        help_text="The target of this user to alternate account relationship",
+    )
+    context = models.TextField(
+        help_text="The reason for why this account was associated as an alt.",
+        max_length=1900
+    )
+    actor = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name='alts_marked',
```

Comment:
Do we need this? I originally added it, but re-reading I think that some form of "look at all the alts I found!" may not be the most interesting feature.

Reply:
What does this do? Allow us to find all the alts that have been assigned by a single user?

Doesn't seem like we need it right now, but could be useful to have

Reply:
Removed it